src/esm_parser/esm_parser.py

```diff
@@ -1489,6 +1489,17 @@ def resolve_basic_choose(config, config_to_replace_in, choose_key, blackdict={})
         else:
             choices_available[ckey] = cval
 
+    # Are choices all booleans?
+    choices_bool = True
```

Comment:
Small suggestion: all_choices_are_bool

Reply:
Apparently I cannot push my commit with your suggestion for some weird reason...